internal/task/manager/getstatus.go

```diff
@@ -0,0 +1,39 @@
+package task
+
+import (
+	"context"
+
+	basegrpc "github.com/mikewurtz/taskman/internal/grpc"
+	basetask "github.com/mikewurtz/taskman/internal/task"
+)
+
+func (tm *TaskManager) GetTaskStatus(ctx context.Context, taskID string) (*Task, error) {
```

Comment:
What's the difference between `GetTaskStatus` and `GetTask`? Both receive a taskID and return a `*Task`.
When should we use one vs the other?

Reply:
getTask is meant to be used for internal library calls while the GetTaskStatus method is meant to be the public function. I am updating the names to be more clear in my next update. I will also simplify GetTaskStatus